build.py

```diff
@@ -27,36 +27,19 @@ def in_cibuildwheel():
     return bool(int(os.environ.get("CIBUILDWHEEL", "0")))
 
 
-def check_venv():
-    # Both conda and venv are configured, pick conda
-    if os.environ.get("CONDA_PREFIX") and os.environ.get("VIRTUAL_ENV"):
-        del os.environ["VIRTUAL_ENV"]
-    # Build with cibuildwheel detected, set `VIRTUAL_ENV`
-    if in_cibuildwheel() and os.environ.get("VIRTUAL_ENV") is None:
-        os.environ["VIRTUAL_ENV"] = os.path.dirname(os.path.dirname(PYTHON_EXECUTABLE_PATH))
-        display(
-            f"Build with cibuildwheel detected, set `VIRTUAL_ENV` as: {os.environ['VIRTUAL_ENV']}"
-        )
-    if not os.environ.get("CONDA_PREFIX") and not os.environ.get("VIRTUAL_ENV"):
-        raise RuntimeError("A virtual env is required to run `maturin develop`")
-
-
 def force_maturin_release() -> bool:
     return os.environ.get("FORCE_MATURIN_RELEASE", "0") == "1"
 
 
 def build():
     run(f"{PYTHON_EXECUTABLE_PATH} --version")
-    run(f"{PYTHON_EXECUTABLE_PATH} -m ensurepip")
-    run(f"{PYTHON_EXECUTABLE_PATH} -m pip freeze")
     run(f"{PYTHON_EXECUTABLE_PATH} misc/generate_pyqt.py")
-    check_venv()
     release = "--release" if in_cibuildwheel() or force_maturin_release() else ""
-    run(f"maturin develop {release}")
-    # An alternative to `maturin develop` is:
-    # run("maturin build -r")
-    # run("unzip -o target/wheels/parsec-*.whl parsec/_parsec.abi3.so")
-    # Note that it does not require a virtualenv, as opposed to `develop`
+    run(f"maturin build {release}")
+    if sys.platform == "win32":
+        run("unzip -o target/wheels/parsec-*.whl parsec/_parsec.pyd")
```

Comment:
I'd rather use python's zip library instead of the system one (otherwise we will soon discover which version of Windows/FooBarLinux is missing this command ^^)

Reply:
implemented by afde298fd1fc4c7eadd89090fb57ffb97ead76e0